share/new_eds/accessor.go

```diff
@@ -16,6 +16,8 @@ type Accessor interface {
 	Size(ctx context.Context) int
 	// DataHash returns data hash of the Accessor.
 	DataHash(ctx context.Context) (share.DataHash, error)
+	// AxisRoots returns axis roots of the Accessor.
```

Comment:
Worth adding a little comment that's also called DAH. This will be useful for people that don't have enough context on the new name.